src/app/design/guidelines/user-assistance/index.html

```diff
@@ -1,19 +1,60 @@
+    </p>
+  </stache-page-summary>
+
+	<stache-page-anchor>
+		Usage
+	</stache-page-anchor>
+	<p>
+		SKY UX design patterns include four primary techniques to display help content: empty-state help, inline help, preloaded Help Widget, and popovers. To determine the appropriate technique for user assistance, consider the complexity of the task and the expected knowledge of users. The chart below provides a matrix to determine which type of user assistance to use in different scenarios. Do not overuse help content because it can overwhelm or inhibit users. For example, only display the preloaded Help Widget and popovers when user research supports the need for it. When in doubt, err on the side of empty-state help and inline help.
+	</p>
+	<div class="screenshot screenshot-info" style="margin-top: 20px;">
+		<img src="~/assets/img/guidelines/userassistance/help-chart.png" style="width: 100%;" />
+	</div>
+
+	<stache-page-anchor>
+		Empty-state help
+	</stache-page-anchor>
+
+	<stache-row>
+		<stache-column screenSmall="6">
+			<div class="sky-subsection-heading">Usage</div>
+			<p>
+				Empty-state help provides user assistance when content containers are not populated. The content can include a title, body copy, and buttons to take actions or access additional resources. The help content is not available when content containers are populated.
+			</p>
+			<div class="sky-subsection-heading">Layout</div>
+				<p>
+					Center empty-state help in its content container.
+				</p>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/userassistance/empty-state-example.png" />
+			</div>
+		</stache-column>
+	</stache-row>
+
 	<stache-page-anchor>
-  	Page- and modal-level help
-  </stache-page-anchor>
+		Inline help
+	</stache-page-anchor>
 
+  <h3 class="sky-subsection-heading">
+  	Page-level and modal-level help
+  </h3>
   <p>
-    When users perform a complex task, they may require comprehensive documentation for an entire task instead of inline help for individual parts of the task. Users should invoke help at the page or modal level, depending on the context. In Blackbaud products, users can invoke the help service from the omnibar or a <a routerLink="/components/modal">modal</a> header bar.
-  </p>  
+    When users perform a complex task, they may require comprehensive documentation for the entire task instead of inline help for individual parts of the task. Users should invoke help at the page or modal level, depending on the context. In Blackbaud products, users can invoke the help service from the omnibar or a <a routerLink="/components/modal">modal</a> header bar.
```

Comment:
Not sure if a link to the bb-help-docs is better suited down below or here. Also, is using the help content specific to Blackbaud developers? I'm never quite sure where that line's drawn. If accessing help content via the help button is BB dev specific, I think it might be worth pointing out because the site's public.

Reply:
Good question. I'm not sure. I'll look into it.